registry/swell/calldata-swell.json

```diff
@@ -0,0 +1,462 @@
+          "outputs": [{ "name": "", "type": "uint256", "internalType": "uint256" }],
+          "stateMutability": "view"
+        },
+        {
+          "type": "function",
+          "name": "totalSupply",
+          "inputs": [],
+          "outputs": [{ "name": "", "type": "uint256", "internalType": "uint256" }],
+          "stateMutability": "view"
+        },
+        {
+          "type": "function",
+          "name": "transfer",
+          "inputs": [
+            { "name": "to", "type": "address", "internalType": "address" },
+            { "name": "amount", "type": "uint256", "internalType": "uint256" }
+          ],
+          "outputs": [{ "name": "", "type": "bool", "internalType": "bool" }],
+          "stateMutability": "nonpayable"
+        },
+        {
+          "type": "function",
+          "name": "transferFrom",
+          "inputs": [
+            { "name": "from", "type": "address", "internalType": "address" },
+            { "name": "to", "type": "address", "internalType": "address" },
+            { "name": "amount", "type": "uint256", "internalType": "uint256" }
+          ],
+          "outputs": [{ "name": "", "type": "bool", "internalType": "bool" }],
+          "stateMutability": "nonpayable"
+        },
+        {
+          "type": "function",
+          "name": "whitelistEnabled",
+          "inputs": [],
+          "outputs": [{ "name": "", "type": "bool", "internalType": "bool" }],
+          "stateMutability": "view"
+        },
+        {
+          "type": "function",
+          "name": "whitelistedAddresses",
+          "inputs": [{ "name": "", "type": "address", "internalType": "address" }],
+          "outputs": [{ "name": "", "type": "bool", "internalType": "bool" }],
+          "stateMutability": "view"
+        },
+        {
+          "type": "function",
+          "name": "withdrawERC20",
+          "inputs": [{ "name": "_token", "type": "address", "internalType": "contract IERC20" }],
+          "outputs": [],
+          "stateMutability": "nonpayable"
+        }
+      ]
+    }
+  },
+  "metadata": {},
+  "display": {
+    "formats": {
+      "addToWhitelist(address)": {
+        "fields": [
+          {
+            "label": "Address",
+            "format": "addressName",
+            "params": { "types": ["wallet", "eoa", "contract", "token", "collection"] },
+            "path": "#._address"
+          }
+        ]
+      },
+      "allowance(address,address)": {
+        "fields": [
+          { "label": "Owner", "format": "addressName", "params": { "types": ["eoa", "wallet"] }, "path": "#.owner" },
+          { "label": "Spender", "format": "addressName", "params": { "types": ["contract"] }, "path": "#.spender" }
+        ]
+      },
+      "approve(address,uint256)": {
+        "fields": [
+          { "label": "Spender", "format": "addressName", "params": { "types": ["contract"] }, "path": "#.spender" },
+          { "label": "Amount", "format": "amount", "path": "#.amount" }
```

Comment:
Is it an amount (in eth) or a token amount? If a token, should be tokenAmount and a path to the token address should be specified